jujugui/static/gui/src/app/components/inspector-config/inspector-config.js

```diff
@@ -117,9 +185,10 @@ YUI.add('inspector-config', function() {
       Generates the list of elements to render for the config UI.
 
       @method _generateConfigElements
+      @param {Object} serviceConfig The current service config.
       @returns {Array} An array of React components.
     */
-    _generateConfigElements: function() {
+    _generateConfigElements: function(serviceConfig) {
```

Comment:
This method doesn't get called outside of the component so instead of passing this in I would have just grabbed the serviceConfig from state...up to you if you wan to change though :)